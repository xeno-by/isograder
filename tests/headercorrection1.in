* Section [/1]
-1 MISTAKE!
