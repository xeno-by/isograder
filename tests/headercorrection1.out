EXIT CODE:
1

STDOUT:

STDERR:
error parsing the correction sheet:
line 2: invalid correction format: expected "[-|+]<number>: <explanation>"
-1 MISTAKE!
^

